aws/table_aws_redshift_cluster.go

```diff
@@ -277,25 +284,26 @@ func tableAwsRedshiftCluster(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Transform:   transform.FromField("Tags"),
 			},
-			// Standard columns
-			{
-				Name:        "tags",
-				Description: resourceInterfaceDescription("tags"),
-				Type:        proto.ColumnType_JSON,
-				Transform:   transform.From(getRedshiftClusterTurbotTags),
-			},
+
+			// Steampipe standard columns
 			{
 				Name:        "title",
 				Description: resourceInterfaceDescription("title"),
 				Type:        proto.ColumnType_STRING,
 				Transform:   transform.FromField("ClusterIdentifier"),
 			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(getRedshiftClusterTurbotTags),
+			},
 			{
 				Name:        "akas",
 				Description: resourceInterfaceDescription("akas"),
 				Type:        proto.ColumnType_JSON,
-				Hydrate:     getRedshiftClusterAkas,
-				Transform:   transform.FromValue(),
+				Hydrate:     getRedshiftClusterARN,
+				Transform:   transform.FromValue().Transform(transform.EnsureStringArray),
```

Comment:
```suggestion
				Transform:   transform.FromField("ClusterNamespaceArn").Transform(transform.EnsureStringArray),
```

I think this should work as well, and doesn't require the separate function